Sources/CompilerPluginSupport/TargetExtensions.swift

```diff
@@ -16,14 +16,14 @@ public extension Target {
     @available(_PackageDescription, introduced: 999.0)
     static func macro(
         name: String,
-        group: TargetGroup,
+        group: TargetGroup?,
```

Comment:
Wouldn't this be better as `group: TargetGroup = .package`, as you did with `target`, `executableTarget`, and `testTarget`?